samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1109,7 +1103,7 @@ mfxStatus CDecodingPipeline::CreateHWDevice()
     }
 
 #if MFX_D3D11_SUPPORT
-    if (D3D11_MEMORY == m_memType)
+    if (D3D9_MEMORY != m_memType)
```

Comment:
same

Reply:
In sample_decode d3d11 device are created for system memory if enabled vpp (-w and -h options), so I left this condition here, otherwise d3d9 device will be created.